docker/api/daemon.py

```diff
@@ -174,3 +174,17 @@ def version(self, api_version=True):
         """
         url = self._url("/version", versioned_api=api_version)
         return self._result(self._get(url), json=True)
+
+    def reload_config(self, dockercfg_path=None):
```

Comment:
Actually, since this is not an API call, it should probably be added to the `APIClient` class instead.